docs/source/bot_methods.rst

```diff
@@ -0,0 +1,291 @@
+.. raw:: html
```

Comment:
```suggestion
.. raw:: html

   <hr>

Since This class has a large number of methods and attributes, below you can find a quick overview.

.. raw:: html
```

 … or some other wording. then you can remove the "Available methods:" for `bot.py`. 
please put the `<hr>` tag also at the end - those make the section a bit more separated from the rest :)